graph/graph.go

```diff
@@ -15,6 +14,8 @@ import (
 	"sync"
 	"time"
 
+	gzip "github.com/klauspost/pgzip"
```

Comment:
Using [github.com/klauspost/compress/gzip](https://github.com/klauspost/compress) is probably better, since we are only compressing small data sizes.